src/NUnitTestAdapter/NUnitTestAdapter.cs

```diff
@@ -136,17 +136,7 @@ protected void Initialize(IDiscoveryContext context, IMessageLogger messageLogge
         protected ITestRunner GetRunnerFor(string assemblyName)
         {
             var package = CreateTestPackage(assemblyName);
-
-            try
-            {
-                return TestEngine.GetRunner(package);
-            }
-            catch (Exception ex)
-            {
-                TestLog.Error("Error: Unable to get runner for this assembly. Check installation, including any extensions.");
-                TestLog.Error(ex.GetType().Name + ": " + ex.Message);
-                throw;
-            }
+            return TestEngine.GetRunner(package);
```

Comment:
It's easy to get this kind of double exception handling if we don't have a test. Maybe we should add a test that shows this method __does__ throw the exception in this situation.